### PR TITLE
[KOGITO-3510] Deploy/Promote pIpelines: set credentials as params

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-3510

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-cloud-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by radtriste** [comment]
Tested with https://github.com/kiegroup/kogito-pipelines/pull/68

**Review by sutaakar — APPROVED** [review]

**Review by Kevin-Mok — APPROVED** [review]

**Review by Kaitou786 — APPROVED** [review]

File: Jenkinsfile.promote [hunk 1]
```diff
@@ -147,7 +147,7 @@ pipeline {
         //             String nextVersion = getNextVersion()
```
Comment (spolti) [inline Jenkinsfile.promote:147]: why is this section commented?

**Review by spolti — APPROVED** [review]

File: Jenkinsfile.promote (hunk 1 above)
Comment (radtriste) [reply Jenkinsfile.promote:147]: `That part is commented as it messes with PRs on operatorhub which may require some changes`

**Comment by radtriste** [comment]
@spolti if you are good with my last comment, then feel free to merge :)

**Pull request merged**